lib/que/job.spec.rb

```diff
@@ -349,6 +383,7 @@ def run(*args)
     def execute(*args)
       TestJobClass.run(*args)
     end
+    ruby2_keywords(:execute) if respond_to?(:ruby2_keywords, true)
```

Comment:
I suspect Job.run needs to be a ruby2_keywords methods and this is obscuring that fact in the tests. Wondering if we can use a let to make these execute methods accept a lambda?

Reply:
Has this been resolved now? I can't find the piece of code you were pointing at anymore

Reply:
Yep, this was fixed in 8f474a36b0791037f81f45734e567781eb36ca7c